site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -37,116 +39,96 @@ The logical relationship between the Gateway API resources looks like this:
+This is a permissive method of Route selection since the Routes are given
+full control to select this Gateway. There are more restrictive forms of Route
+selection that allow selection on a per-Namespace basis, detailed
 in [Route binding](/concepts/api-overview/#route-binding). The following block
-specifies how this Gateway allows HTTPRoutes from all Namespaces in the 
+specifies how this Gateway allows HTTPRoutes from all Namespaces in the
 cluster to bind to it:
 
 ```yaml
     routes:
-      kind: HTTPRoute
       namespaces:
         from: "All"
+      kinds:
+      - HTTPRoute
 ```
 
-Meanwhile, the store team deploys their route for the `store` Service in the 
+Meanwhile, the store team deploys their route for the `store` Service in the
 `store-ns` Namespace:
 
 ```yaml
-{% include 'v1alpha1/cross-namespace-routing/store-route.yaml' %}
-```
-
-This Route has straightforward routing logic as it just matches for 
-`/store` traffic which it sends to the `store` Service. The following snippet 
-of the [`gateways` field](/v1alpha2/references/spec/#networking.x-k8s.io/v1alpha1.RouteGateways) 
-controls which Gateways this Route can bind to:
-
-```yaml
-  gateways:
-    allow: FromList
-    gatewayRefs:
-    - name: shared-gateway
-      namespace: infra
+{% include 'v1alpha2/cross-namespace-routing/store-route.yaml' %}
 ```
 
-`gateways.allow` can be configured for Gateways in the same Namespace as the
-Route (the default), all Gateways, or a list of specific Gateways. In this
-example the store and site teams decide to reference a specific Gateway. This is
-the least permissive choice which ensures that other Gateways in the cluster
-(perhaps created in the future at some point) will not bind with these Routes.
-If cluster administrators have full control over how Gateways are deployed in a
-cluster then a more permissive binding option could be configured on Routes. The
-less permissive the Gateway selection is, the less that application owners need
-to know about which Gateways are deployed. 
+This Route has straightforward routing logic as it just matches for
+`/store` traffic which it sends to the `store` Service.
 
 The site team now deploys Routes for their applications. They deploy two
 HTTPRoutes into the `site-ns` Namespace:
 
-- The `home` HTTPRoute acts as a default routing rule, matching for all traffic 
-to `foo.example.com/*` not matched by an existing routing rule and sending it to 
+- The `home` HTTPRoute acts as a default routing rule, matching for all traffic
+to `foo.example.com/*` not matched by an existing routing rule and sending it to
 the `home` Service.
-- The `login` HTTPRoute  routes traffic for `foo.example.com/login` to 
-`service/login-v1` and `service/login-v2`. It uses weights to granularly 
-control traffic distribution between them. 
+- The `login` HTTPRoute  routes traffic for `foo.example.com/login` to
+`service/login-v1` and `service/login-v2`. It uses weights to granularly
```

Comment:
```suggestion
- The `login` HTTPRoute  splits traffic to `foo.example.com/login` between
`service/login-v1` and `service/login-v2`. It uses weights to granularly
```

Reply:
this text was significantly changed in 8908617